lib/rtcpeerconnection/chrome.js

```diff
@@ -115,12 +122,36 @@ ChromeRTCPeerConnection.prototype.addStream = function addStream(stream) {
     return;
   }
   this._localStreams.add(stream);
-  this._peerConnection.addStream(stream);
+
+  if (!RtpSender) {
```

Comment:
I think it might be better to check for the presence of `addTrack` on the native `prototype`.